modules/svg/SCsub

```diff
@@ -11,6 +11,16 @@ thirdparty_obj = []
 
 thirdparty_dir = "#thirdparty/thorvg/"
 thirdparty_sources = [
+    "src/lib/sw_engine/tvgSwFill.cpp",
+    "src/lib/sw_engine/tvgSwImage.cpp",
+    "src/lib/sw_engine/tvgSwMath.cpp",
+    "src/lib/sw_engine/tvgSwMemPool.cpp",
+    "src/lib/sw_engine/tvgSwRaster.cpp",
+    "src/lib/sw_engine/tvgSwRenderer.cpp",
+    "src/lib/sw_engine/tvgSwRle.cpp",
+    "src/lib/sw_engine/tvgSwShape.cpp",
+    "src/lib/sw_engine/tvgSwStroke.cpp",
+    "src/lib/tvgAccessor.cpp",
```

Comment:
This was the only file missing in our build. It doesn't seem to actually be used for anything in upstream code, but I compiled it anyway in case they start using it in future updates.